src/module_3/inference.py

```diff
@@ -0,0 +1,31 @@
+
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+import os
+from joblib import load
+
+from sklearn.linear_model import LogisticRegression
+
+from general_functions import prepare_data,train_val_split, feature_target_split, standardise_data, evaluate_model
+
+OUTPUT_PATH = Path("/mnt/c/Users/Adriana/Desktop/ZRIVE/zrive-ds/src/module_3/out")
+DATA_PATH = Path("/mnt/c/Users/Adriana/Desktop/ZRIVE/data/groceries/box_builder_dataset/feature_frame.csv")
+FEATURE_COLS = ['ordered_before', 'global_popularity', 'abandoned_before', 'normalised_price']
+
+
+def main():
```

Comment:
You are just inferencing the test data. Something that is much closer to what you would implement in real life would be a class / script that accepts input data, make the inference and returns it.

This script is more a "evaluation" step where you double check the metrics of some validation data